src/blocks/mrc_steps.ts

```diff
@@ -0,0 +1,285 @@
+/**
+ * @license
+ * Copyright 2025 Porpoiseful LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Blocks for class method definition
+ * @author alan@porpoiseful.com (Alan Smith)
+ */
+import * as Blockly from 'blockly';
+import { Order } from 'blockly/python';
+
+import { MRC_STYLE_STEPS } from '../themes/styles';
+import { ExtendedPythonGenerator } from '../editor/extended_python_generator';
+import { createStepFieldFlydown } from '../fields/field_flydown';
+import { BLOCK_NAME as MRC_JUMP_TO_STEP } from './mrc_jump_to_step';
+import * as stepContainer from './mrc_step_container'
+
+export const BLOCK_NAME = 'mrc_steps';
+
+/** Extra state for serialising mrc_steps blocks. */
+type StepsExtraState = {
+  /**
+   * The steps
+   */
+  stepNames: string[],
+};
+
+export type StepsBlock = Blockly.Block & StepsMixin & Blockly.BlockSvg;
+interface StepsMixin extends StepsMixinType {
+  mrcStepNames: string[];
+}
+type StepsMixinType = typeof STEPS;
+
+const STEPS = {
+  /**
+   * Block initialization.
+   */
+  init: function (this: StepsBlock): void {
+    this.mrcStepNames = ["0"];
```

Comment:
Initializing mrcStepNames here to ["0"] is causing the block to move when the opmode module is loaded into the editor. I think because of the "0" step being removed and the actual steps from extraState being added.

To prevent that, you should set mrcStepNames here to [].

Then, the block for the toolbox should have extraState with stepNames ['0'].